lab_1_classify_by_unigrams/main.py

```diff
@@ -88,20 +176,46 @@ def collect_profiles(paths_to_profiles: list) -> list[dict[str, str | dict[str,
     :paths_to_profiles: a list of strings to the profiles
     :return: a list of loaded profiles
     """
+    if isinstance(paths_to_profiles, list):
+        preprocess_profiles_list = []
+        for path in paths_to_profiles:
+            preprocess_profiles_list.append(preprocess_profile(load_profile(path)))
+        return preprocess_profiles_list
+
+    return None
 
 
-def detect_language_advanced(unknown_profile: dict[str, str | dict[str, float]],
-                             known_profiles: list) -> list | None:
+def detect_language_advanced(
```

Comment:
why did you change it?

Reply:
Because code style check didn't like the length of the line)

Reply:
It is strange. What exactly style checker? On CI?

Reply:
Yes, on CI

Reply:
Please, provide a proof - link to failed CI

Reply:
![tg_image_1220455557](https://github.com/fipl-hse/2023-2-level-labs/assets/116805511/91ebec52-01dd-446b-8b3e-f6b6362ad8a4)

Reply:
In my code lines 99 and 197 are two functions: detect_language and detect_language_advanced. That's two lines that I changed because of code style errors.

Reply:
Yes